src/Nest/Analysis/TokenFilters/HunspellTokenFilter.cs

```diff
@@ -12,6 +13,7 @@ public interface IHunspellTokenFilter : ITokenFilter
 		/// If true, dictionary matching will be case insensitive.
 		/// </summary>
 		[JsonProperty("ignore_case")]
+		[Obsolete("Scheduled to be removed in 6.0")]
```

Comment:
Is it scheduled still for a later Elasticsearch 6.x, or been removed in Elasticsearch 6.0? Can we advise an alternative approach e.g. using a `lowercase` token filter before `hunspell` token filter?

Reply:
Added help and changed to 6.x.